PKGBUILD

```diff
@@ -1,56 +1,49 @@
 # Maintainer: Chris Salzberg <chris@dejimata.com>
-# Contributor: Leonidas Spyropoulos <artafinde@gmail.com>
 
-pkgname=neomutt
-pkgver=20170912
+_pkgname=neomutt
+pkgname=neomutt-git
+pkgver=neomutt.20171215.r0.gae612851
```

Comment:
Shouldn't this be `latest`, since the `pkgver` is actually calculated by the function below?

Reply:
I apologise for missing that before committing. Thank you for catching my mistake.

Reply:
np, thank you for jumping into this 😄